pre-commit/strip.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# strip all TARGET string what matches PATTERN
+# sh strip.sh abcb b -> ab
```

Comment:
This comment is inaccurate given the code. :)
It's also a little weird that the example uses sh when the bang path is bash, and the code is not POSIX sh friendly.
```suggestion
# bash strip.sh abcb b -> ac
```